doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p> change to :</p><p>"First let's bring up JupySQL and connect to the database via SQLAlchemy. Please refer to the documentation of <a href="https://github.com/exasol/sqlalchemy-exasol" target="_blank">sqlalchemy-exasol</a> for details on how to connect to the database using Exasols SQLAlchemy driver."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>please also take a look at my phrasing related suggestions.</p>

Reply:
<p>"using Exasol<strong>s</strong> SQLAlchemy driver."</p>